config/connector_campaign_finance.config

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "datagen-campaign_finance",
+  "config": {
+    "connector.class": "io.confluent.kafka.connect.datagen.DatagenConnector",
+    "kafka.topic": "campaign_finance",
+    "quickstart": "campaign_finance",
+    "key.converter": "org.apache.kafka.connect.storage.StringConverter",
```

Comment:
need a `schema.keyfield=cand.id` here?

Reply:
sure. Was asking as the other pr had it mentioned explicitly https://github.com/confluentinc/kafka-connect-datagen/pull/106